VirtoCommerce.MarketingModule.Web/Scripts/promotion/blades/catalog-entry-list.js

```diff
@@ -0,0 +1,154 @@
+                    template: 'Modules/$(VirtoCommerce.Catalog)/Scripts/blades/common/catalog-items-select.tpl.html',
+                    breadcrumbs: [],
+                    toolbarCommands: [
+                    {
+                        name: "platform.commands.pick-selected", icon: 'fa fa-plus',
+                        executeMethod: function () {
+                            blade.$scope.gridApi.grid.options.data = _.map(selectedListEntries, function (entry) {
+                                if (entry.imageUrl) {
+                                    entry.imgSrc = entry.imageUrl;
+                                }
+                                return entry;
+                            });
+                            bladeNavigationService.closeBlade(catalogBlade);
+                        },
+                        canExecuteMethod: function () {
+                            return selectedListEntries.length > 0;
+                        }
+                    }]
+                };
+
+                catalogBlade.options = {
+                    selectedItemIds: blade.productIds,
+                    showCheckingMultiple: false,
+                    checkItemFn: function (listItem, isSelected) {
+                        if (listItem.type == 'category') {
+                            blade.error = 'Must select Product';
+                            listItem.selected = undefined;
+                        } else {
+                            if (isSelected) {
+                                if (_.all(selectedListEntries, function (x) { return x.id != listItem.id; })) {
+                                    selectedListEntries.push(listItem);
+                                }
+                            }
+                            else {
+                                selectedListEntries = _.reject(selectedListEntries, function (x) { return x.id == listItem.id; });
+                            }
+                            blade.error = undefined;
+                        }
+                    }
+                };
+
+                bladeNavigationService.showBlade(catalogBlade, blade);
+            }
+
+            $scope.blade.toolbarCommands = [
+                {
+                    name: "platform.commands.refresh", icon: 'fa fa-refresh',
+                    executeMethod: function (blade) {
+                        blade.refresh();
+                    },
+                    canExecuteMethod: function () { return true; }
+                },
+                {
+                    name: "platform.commands.add", icon: 'fa fa-plus',
+                    executeMethod: function () {
+                        openCatalogBlade();
+                    },
+                    canExecuteMethod: function () {
+                        return true;
+                    }
+                },
+                {
+                    name: "platform.commands.delete", icon: 'fa fa-trash-o',
```

Comment:
See how to delete method implemented in marketing module https://github.com/VirtoCommerce/vc-module-marketing/blob/dev/VirtoCommerce.MarketingModule.Web/Scripts/promotion/blades/promotion-list.js#L44

Reply:
Done